tests/core/filtering/test_contract_on_event_filtering.py

```diff
@@ -204,10 +204,21 @@ def test_on_sync_filter_with_topic_filter_options_on_old_apis(
     seen_logs = event_filter.get_new_entries()
     assert len(seen_logs) == 4
 
-    post_event_filter = contract.events.LogTripleWithIndex.createFilter(
+    post_event_filter = contract.events.LogTripleWithIndex.create_filter(
         argument_filters={'arg1': [1, 2], 'arg2': [1, 2]},
         fromBlock=0,
     )
 
     old_logs = post_event_filter.get_all_entries()
     assert len(old_logs) == 4
+
+
+#
+# Deprecated
+#
+def test_createFilter_emits_deprecation_warning(web3, emitter):
```

Comment:
It might be nice, as a gut check, to check that `createFilter()` actually works and / or delegates to `create_filter()`. Since this is a pretty minor addition and we don't expect any future `v5` releases, I think it is easy enough to see that it's working as intended so feel free to take or leave.

Reply:
That's a good point. Will add the expectations back!